.circleci/config.yml

```diff
@@ -3,9 +3,10 @@ version: 2
 jobs:
   test:
     docker:
-      - image: cimg/node:16.14.2
+      - image: cimg/node:18.18.0
     steps:
       - checkout
+      - run: yarn --prod # install only prod dependencies first to trigger postinstall patch-package script
```

Comment:
I believe the reason you have to resort to this hack is because of this:
https://github.com/ds300/patch-package/issues/84#issuecomment-461221012

We should make sure we undo this as we work on 2.x branch.